### PR TITLE
Add more keywords/attributes to syntax highlighting

### PR DESCRIPTION
**Pull request by stephencelis**
Noticed that a bunch of things weren't syntax highlighting nicely in the latest transcript.

### CODE REVIEW

File: Sources/Styleguide/PointFreeBaseStyles.swift [hunk 1]
```diff
@@ -429,6 +429,7 @@ private let inlineCodeStyles =
 private let token = CssSelector.class("token")
 
 private let prismJsTheme: Stylesheet = concat([
+  (token & CssSelector.class("atrule")) % color(.other("#d73a49")),
```
Comment (stephencelis) [inline Sources/Styleguide/PointFreeBaseStyles.swift:432]: Adds a rule to syntax highlight attributes as keywords.

File: Sources/Views/PageLayout.swift [hunk 2]
```diff
@@ -282,10 +282,10 @@ Prism.languages.swift = Prism.languages.extend('clike', {
...
+  'keyword': /\b(?:actor|Any|as|associativity|associatedtype|async|await|break|case|catch|class|continue|convenience|default|defer|deinit|didSet|do|dynamic(?:Type)?|else|enum|extension|fallthrough|false|fileprivate|final|for|func|get|guard|if|import|in|indirect|infix|init|inout|internal|is|lazy|left|let|mutating|new|nil|none|nonisolated|nonmutating|open|operator|optional|override|postfix|precedence|precedencegroup|prefix|private|protocol|Protocol|public|repeat|required|rethrows|return|right|safe|self|Self|set|some|static|struct|subscript|super|switch|throws?|true|try|Type|typealias|unowned|unsafe|var|weak|where|while|willSet|_(?:modify|read)?|__(?:COLUMN__|FILE__|FUNCTION__|LINE__)|#(?:available|colorLiteral|column|dsohandle|elseif|else|endif|error|fileID|fileLiteral|filePath|file|function|if|imageLiteral|keyPath|line|selector|sourceLocation|warning))\b/,
```
Comment (stephencelis) [inline Sources/Views/PageLayout.swift:285]: We were missing `associatedtype` somehow, so added it along with a bunch of others.

File: Sources/Views/PageLayout.swift [hunk 3]
```diff
@@ -282,10 +282,10 @@ Prism.languages.swift = Prism.languages.extend('clike', {
...
   'number': /\b(?:[\d_]+(?:\.[\de_]+)?|0x[a-f0-9_]+(?:\.[a-f0-9p_]+)?|0b[01_]+|0o[0-7_]+)\b/i,
   'constant': /\b(?:nil|[A-Z_]{2,}|k[A-Z][A-Za-z_]+)\b/,
-  'atrule': /@\b(?:IB(?:Outlet|Designable|Action|Inspectable)|class_protocol|exported|noreturn|NS(?:Copying|Managed)|objc|UIApplicationMain|auto_closure)\b/,
+  'atrule': /@\b(?:available|auto_closure|autoclosure|class_protocol|convention|discardableResult|dynamicCallable|dynamicMemberLookup|escaping|exported|frozen|GKInspectable|IB(?:Outlet|Designable|Action|Inspectable)|inlinable|inline|main|noreturn|NS(?:ApplicationMain|Copying|Managed)|objc|objcMembers|propertyWrapper|requires_stored_property_inits|resultBuilder|rethrows|testable|UIApplicationMain|unknown|usableFromInline|warn_unqualified_access|_(?:disfavoredOverload|exported))\b/,
```
Comment (stephencelis) [inline Sources/Views/PageLayout.swift:288]: Makes `@inlinable`, `@usableFromInline` and others highlight nicely.

**Review by mbrandonw — APPROVED** [review]
nice

**Pull request merged**